$(var namespace)/MWFCN_node:
  ros__parameters:
    map_topic: /$(var namespace)/map
    costmap_topic: /$(var namespace)/global_costmap/costmap
    robot_base_frame: base_footprint                            # Without namespace (prefix)
    robot_frame_prefix: ssugv                                   # robot is the prefix where frames are s.t. robot1/base_footprint, robot2/base_footprint

    rate: 0.3

    obstacle_inflation_radius: 0.3
    robot_count: 3


